connaisseur/validators/notaryv1/notaryv1_validator.py

```diff
@@ -111,8 +115,13 @@ def __process_chain_of_trust(
         tuf_roles = ["root", "snapshot", "timestamp", "targets"]
 
         # load all trust data
-        for role in tuf_roles:
-            trust_data[role] = self.notary.get_trust_data(image, TUFRole(role))
+        t1 = dt.datetime.now()
+        trust_data_list = await asyncio.gather(
+            *[self.notary.get_trust_data(image, TUFRole(role)) for role in tuf_roles]
+        )
+        t2 = (dt.datetime.now() - t1).total_seconds()
+        logging.debug(f"TIMER: {t2}")
```

Comment:
If you want to leave the debug stmt in here, I'd make it more specific: `Pulled trust data for image {image} in {t2} seconds` or the like